x-pack/plugins/ml/public/cases/register_anomaly_swim_lane_attachment.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButtonIcon, EuiDescriptionList } from '@elastic/eui';
+import { CasesUiSetup } from '@kbn/cases-plugin/public';
+import { CoreStart } from '@kbn/core/public';
+import { PersistableStateAttachmentViewProps } from '@kbn/cases-plugin/public/client/attachment_framework/types';
+import { FIELD_FORMAT_IDS } from '@kbn/field-formats-plugin/common';
+import { MlStartDependencies } from '../plugin';
+import { ANOMALY_SWIMLANE_EMBEDDABLE_TYPE, AnomalySwimlaneEmbeddableInput } from '..';
+import { PLUGIN_ICON } from '../../common/constants/app';
+import { getAnomalySwimLaneEmbeddableComponent } from '../embeddables/anomaly_swimlane';
+
+const AttachmentActions: React.FC = () => {
+  return (
+    <EuiButtonIcon
+      data-test-subj="test-attachment-action"
+      onClick={() => {}}
+      iconType="boxesHorizontal"
+      aria-label="See attachment"
+    />
+  );
+};
+
+export function registerAnomalySwimLaneCasesAttachment(
+  cases: CasesUiSetup,
+  coreStart: CoreStart,
+  pluginStart: MlStartDependencies
+) {
+  cases.attachmentFramework.registerPersistableState({
+    id: ANOMALY_SWIMLANE_EMBEDDABLE_TYPE,
+    icon: PLUGIN_ICON,
+    // TODO check where this name is presented
+    displayName: i18n.translate('xpack.ml.cases.anomalySwimLane.displayName', {
+      defaultMessage: 'Anomaly swim lane',
+    }),
+    getAttachmentViewObject: () => ({
+      event: (
+        <FormattedMessage
+          id="xpack.ml.cases.anomalySwimLane.embeddableAddedEvent"
+          defaultMessage="added the Anomaly Swim Lane embeddable"
```

Comment:
I'm not sure how and where this message is presented so I highlight that you might want to change it to sentence-case like one of the following options. Please ignore this comment if it's not applicable based on the context. 

```suggestion
          defaultMessage="Added the Anomaly Swim Lane embeddable"
```

or 

```suggestion
          defaultMessage="Anomaly Swim Lane embeddable has been added"
```

Reply:
<img width="707" alt="image" src="https://user-images.githubusercontent.com/5236598/186159956-c99cfc32-9b36-45d4-b882-c268f50b3f72.png">

@szabosteve it's rendered in a context of the attachment action

Reply:
@darnautov Awesome, thank you for the explanation. My comment is not relevant then.